tests/postgres_tests/__init__.py

```diff
@@ -14,3 +20,25 @@ def tearDownClass(cls):
 
         connection_created.disconnect(register_hstore_handler)
         super().tearDownClass()
+
+
+# To locate the widget's template.
+@modify_settings(INSTALLED_APPS={'append': 'django.contrib.postgres'})
+class WidgetTestCase(SimpleTestCase):
```

Comment:
This is copied from `tests/forms_tests/widget_tests/base.py` and should be moved to `django/test/utils.py` (or something) so it can be shared, I suppose.